pkg/webhook/pod/pods.go

```diff
@@ -192,8 +181,52 @@ func (pc *PodAdmissionControl) admitDeletePods(name, namespace string) *v1beta1.
 	if l.IsPD() {
 		pdClient := pc.pdControl.GetPDClient(pdapi.Namespace(namespace), tcName, tc.Spec.EnableTLSCluster)
 		return pc.admitDeletePdPods(pod, ownerStatefulSet, tc, pdClient)
+	} else if l.IsTiKV() {
+		pdClient := pc.pdControl.GetPDClient(pdapi.Namespace(namespace), tcName, tc.Spec.EnableTLSCluster)
+		return pc.admitDeleteTiKVPods(pod, ownerStatefulSet, tc, pdClient)
 	}
 
 	klog.Infof("[%s/%s] is admit to be deleted", namespace, name)
 	return util.ARSuccess()
 }
+
+// Webhook server receive request to create pod
+// if this pod wasn't member of tidbcluster, just let the request pass.
+// Currently we only check with tikv pod
+func (pc *PodAdmissionControl) AdmitCreatePods(ar admission.AdmissionReview) *admission.AdmissionResponse {
+	pod := &core.Pod{}
+	if err := json.Unmarshal(ar.Request.Object.Raw, pod); err != nil {
+		klog.Errorf("Could not unmarshal raw object: %v", err)
+		return util.ARFail(err)
+	}
+
+	name := pod.Name
+	namespace := pod.Namespace
+
+	klog.Infof("receive admission to %s pod[%s/%s]", "create", namespace, name)
+
+	l := label.Label(pod.Labels)
+	if !(l.IsPD() || l.IsTiKV() || l.IsTiDB()) {
+		klog.Infof("pod[%s/%s] is not TiDB component,admit to create", namespace, name)
+		return util.ARSuccess()
+	}
+
+	tcName, exist := pod.Labels[label.InstanceLabelKey]
```

Comment:
Why fail on this when there is no instance label? If a pod does not have the instance label, it may not be managed by tidb-operator.

Reply:
Updated. 
For now, the pod which is going to be created or deleted would be checked whether is managed by `tidb-operator` and is pd/tikv/tidb first. If not, the request would be admitted.

After that, if the pod don't have `InstanceLabelKey`, the request for creating would be refused while the request for deleting would be admitted.